app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="flex_d navigation py-6 px-3 blue-nav-white-text ">
+  <ul class="display_nav flex justify-between items-center ">
+    <li class="w-10"><%= link_to image_tag('backarrow.png', alt: 'Back'), categories_path, class: 'nav-link' %></li>
+    <h2><strong>New purchase</strong></h2>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "nav-link text-green" %></li>
+  </ul>
+<div class="mt-10 ml-12 flex flex-col gap-5 ms-4 justify-center content-center  flex-grow">
+<%= render "shared/form_purchase", attributes: @attributes %>
+ | <%= link_to "home purchase", purchases_path,class:"text-green" %>
+</div>
+</div>
```

Comment:
- [ ] Kindly try as much as you can to make this page look like the given design. The button should be as big as the add categories button and should be positioned at the bottom. Pay attention to spaces for the purchases and the button.


![button](https://github.com/Kel-nana/Budget-app/assets/53124548/aced4b38-9778-4019-8dda-24a89bd5f3f5)


![btn](https://github.com/Kel-nana/Budget-app/assets/53124548/ef83dc22-d18d-4c24-8720-24541deb5598)